src/Classes/iTones/iTones_TrackRequest.php

```diff
@@ -93,7 +93,8 @@ private function canRequestTrack()
      */
     private function trackCanBePlayed()
     {
-        return !(MyRadio_TracklistItem::getIfPlayedRecently($this->track));
+        return (!(MyRadio_TracklistItem::getIfPlayedRecently($this->track)) &&
+               ($this->track->getClean() !== 'n'));
```

Comment:
excessive number of brackets?

` return !MyRadio_TracklistItem::getIfPlayedRecently($this->track) && $this->track->getClean() !== 'n';` ?